File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int userInput = get_int("Height: ");

    while (userInput < 1 || userInput > 8)
    {
        userInput = get_int("Height: ");
    }

    for (int i = 1; i <= userInput; i++)
    {
        for (int c = 0; c < userInput - i; c++)
        {
            printf(" ");
        }

        for (int z = 0; z < i; z++)
        {
            printf("#");
        }

        printf("  ");

        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
while True:
    try:
        userInput = int(input("Height: "))
        if userInput >= 1 and userInput <= 8:
            break

    except ValueError:
        pass

for i in range(1, userInput + 1):
    for j in range(userInput - i):
        print(" ", end="")

    for z in range(0, i):
        print("#", end="")

    print("  ", end="")

    for x in range(0, i):
        print("#", end="")

    print()
